config/environment.js

```diff
@@ -62,6 +61,33 @@ module.exports = function (environment) {
     },
   };
 
+  if (process.env.SENTRY_DSN) {
+    ENV["@sentry/ember"] = {
+      // Will silence Ember.onError warning without the need of using Ember debugging tools.
+      ignoreEmberOnErrorWarning: false,
+
+      // All runloop queue durations will be added as spans.
+      minimumRunloopQueueDuration: 0,
+
+      // Will disable automatic instrumentation for components.
+      // disableInstrumentComponents: true,
+
+      // All (non-glimmer) component render durations will be added as spans.
+      minimumComponentRenderDuration: 0,
+
+      // All component definitions will be added as spans.
+      enableComponentDefinition: true,
+      sentry: {
+        environment,
+        dsn: process.env.SENTRY_DSN,
+        debug: environment !== "production",
+        tracesSampleRate: 0.01,
+        maxBreadcrumbs: 20,
+      },
```

Comment:
Delete this stuff here, since you added it already in the `Sentry.init(...)` call.

Reply:
Or let's simply spread the `sentryConfig` object in the `Sentry.init(...)` call. Then you would have to delete the duplicated entries from the call above.